README.md

```diff
@@ -1,13 +1,67 @@
-##Boilerplate app for Electron with Redux/React
-
-- ES6/ES7 via babel
-- BrowserSync for livereload
-- structure for redux/react app
-- scripts for building cross-platform releases
-
-####Commands
-- `npm run serve` - Start electron app in development
-- `npm run pack` - Create a package for distribution for macOS, Windows and Linux.
-- `npm run pack:mac` - Create a package for distribution for macOS only.
-- `npm run pack:win` - Create a package for distribution for Windows only.
-- `npm run pack:linux` - Create a package for distribution for Linux only.
+# electron-react-redux-boilerplate
+
+Minimal boilerplate to get started with [Electron](http://electron.atom.io/), [React](https://facebook.github.io/react/) and [Redux](http://redux.js.org/).
+
+What's included:
+
+* [React Router](https://reacttraining.com/react-router/)
+* [Redux Thunk](https://github.com/gaearon/redux-thunk/)
+* [Redux Actions](https://github.com/acdlite/redux-actions/)
+* [Redux Local Storage](https://github.com/elgerlambert/redux-localstorage/)
+* [Electron Packager](https://github.com/electron-userland/electron-packager)
+* [Electron DevTools Installer](https://github.com/bradstewart/electron-devtools-installer)
+* [Mocha](https://mochajs.org/)
+
+## Quick start
+
+Clone the repository
+```bash
+git clone git@github.com:jschr/electron-react-redux-boilerplate.git
+```
+
+Install dependencies
+```bash
+cd electron-react-redux-boilerplate
+npm install
+```
+
+Development
+```bash
+npm run serve
```

Comment:
`serve` was popular long ago. I like `npm run develop` as it fully describes what is going on and less misleading.